src/versionUtils.ts

```diff
@@ -18,30 +18,34 @@
 import * as vscode from 'vscode';
 import * as extension from './extension';
 import { Errorable, failed } from './errorable';
-import * as downloader from './downloader';
 import * as config from './config';
 import * as kamelCli from './kamel';
-import * as fs from 'fs';
-import {platformString, kamelUnavailableRejection} from './installer';
+import { platformString, kamelUnavailableRejection } from './installer';
 import fetch from 'cross-fetch';
 
-export const version : string = '1.0.0-RC2'; //need to retrieve this if possible, but have a default
+export const version: string = '1.0.0-RC2'; //need to retrieve this if possible, but have a default
+/*
+* Can be retrieved using `curl -i https://api.github.com/repos/apache/camel-k/releases/latest` and searchign for "last-modified" attribute
+* To be updated when updating the default "version" attribute
+*/
+const LAST_MODIFIED_DATE_OF_DEFAULT_VERSION: string = 'Fri, 28 Feb 2020 07:24:17 GMT';
+let latestVersionFromOnline: string;
 
-async function testVersionAvailable(versionToUse : string): Promise<boolean> {
-	return new Promise<boolean>( async (resolve) => {
+async function testVersionAvailable(versionToUse: string): Promise<boolean> {
```

Comment:
if you return a Promise this function is automatically async. There is no real need to specify it twice. Same applies you you don't specify a return value, then async will make it a Promise automatically.

Reply:
it is not part of the modification that I made.
the change doesn't seem trivial for me so I prefer tol keep it as-is in this PR